md/helptext_columns_water.md

```diff
@@ -1 +1,3 @@
-Processes can consume or produce water. It is assumed that water demand of one process can be provided by excess water by other processes along the chain. Surplus excess water is assumed to be wasted. water demand that cannot be covered by excess heat is either bought at specific water prices, which can be adjusted in the **specific costs** section, or produced by water desalination plants. For the selected supply region, you can choose between these options in the sidebar.
+Processes can consume or produce water. It is assumed that water demand of one process can be provided by excess water by other processes along the chain. Surplus excess water is assumed to be wasted. Water demand that cannot be covered by excess heat is either bought at specific water prices, which can be adjusted in the **specific costs** section, or produced by water desalination plants. For the selected supply region, you can choose between these options in the sidebar.
+
+**Unit:** kg per main output
```

Comment:
@markushal please check: this was "kWh per main output" in the old column description but seems to be more correct with kg

Reply:
unit of main output can be kg or MWh